src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -548,7 +548,7 @@ See the accompanying LICENSE file for applicable license.
     <xsl:choose>
       <xsl:when test="@othertype">
         <xsl:apply-templates select="." mode="process.note.common-processing">
-          <xsl:with-param name="type" select="'note'"/>
+          <xsl:with-param name="type" select="replace(normalize-space(@othertype), ' ', '_')"/>
```

Comment:
This will not work because the value of `$type` is used to look up a string variable and if the source has `othertype="cat"`, there will likely not be a string variable with ID `Cat`. A plug-in can add one, but if one is not added, the note comes out with out a label or defaults to `note` in every language.